src/game_constants.h

```diff
@@ -97,4 +97,10 @@ constexpr double accuracy_goodhit  = 0.5;
 constexpr double accuracy_standard = 0.8;
 constexpr double accuracy_grazing  = 1.0;
 
+/** Minimum item damage output of relevant type to allow using with relevant weapon skill */
+#define MELEE_STAT 5
```

Comment:
I've updated that retaining the constant only so that `item_factory.cpp` assigns the weapon category consistently with the check in `is_melee()`